views/fe-utils/siteTooltipContent.js

```diff
@@ -0,0 +1,30 @@
+import { TOTAL_LABEL, NA } from '../../constants/vars'
+import { formatAsPercentage } from './formatAsPercentageUtil'
+
+export const siteTooltipContent = (siteData) => {
+  const tooltipData = []
+  let siteTotalCount = 0
+  let siteTotalPercent = 0
```

Comment:
do we not get these values from the `studyTotals` or some other key in the site data?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/4a43fd4984d56defdb4112250a883f89a70016a2

Reply:
it looks like there is still a variable that we can get from study totals.